### PR TITLE
Implement a loader for rackspace cloudfiles

### PR DESCRIPTION
**Pull request by BradWhittington**
Hi,

We need a loader for pulling files off rackspace, and I've extended your library to include both a loader and a storage.

### CODE REVIEW

File: thumbor_rackspace/__init__.py [hunk 1]
```diff
@@ -16,4 +16,12 @@
...
 Config.define('RACKSPACE_RESULT_STORAGES_CONTAINER_ROOT', '/products/', 'Root storage path', 'RESULT_STORAGE')
 Config.define('RACKSPACE_RESULT_STORAGE_EXPIRES', True, 'Whether to set expires headers on the cloudfiles objects', 'RESULT_STORAGE')
-__version__ = "0.3"
+
+Config.define('RACKSPACE_PYRAX_CFG', '~/.pyrax.cfg', 'Pyrax configuration file location', 'LOADER')
+Config.define('RACKSPACE_PYRAX_IDENTITY_TYPE', 'rackspace', 'Pyrax identity_type', 'LOADER')
+Config.define('RACKSPACE_PYRAX_REGION', 'SYD', 'Whether to connect to the private rackspace network', 'LOADER')
+Config.define('RACKSPACE_PYRAX_PUBLIC', True, 'Whether to connect to the private rackspace network', 'LOADER')
+Config.define('RACKSPACE_LOADER_CONTAINER', 'thumbor-test', 'Rackspace cloudfiles container', 'LOADER')
+Config.define('RACKSPACE_LOADER_CONTAINER_ROOT', '/products/', 'Root storage path', 'LOADER')
```
Comment (CodingNinja) [inline thumbor_rackspace/__init__.py]: Could you set this to be an empty string so it's more generic for other use cases, please?

**Comment by CodingNinja** [comment]
Nice work, happy to merge once that default value is fixed up :+1:

File: thumbor_rackspace/__init__.py (hunk 1 above)
Comment (BradWhittington) [inline thumbor_rackspace/__init__.py]: Sure, I just followed the same set up as RACKSPACE_RESULT_STORAGES_CONTAINER_ROOT

Should I set both to '' ?

**Comment by BradWhittington** [comment]
Set defaults to ''

**Comment by BradWhittington** [comment]
@CodingNinja this should be good now...

**Comment by CodingNinja** [comment]
@BradWhittington Sorry about that mate, I thought I had merged this already. Appreciate the contribution!

**Pull request merged**